src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -230,8 +230,9 @@ public function fill(array $attributes)
             if ($this->isFillable($key)) {
                 $this->setAttribute($key, $value);
             } elseif ($totallyGuarded) {
+                $model = get_class($this);
```

Comment:
You can use `static::class` instead, removes the overhead of a function call

Reply:
Ok thanks, updated!